content/tutorials/portenta-h7/por-ard-kvs/content.md

```diff
@@ -26,6 +26,9 @@ Let's program the Portenta with a sketch. We will also define a few helper funct
 * Create a new sketch named `FlashKeyValue.ino` 
 * Create a new file named `FlashIAPLimits.h` to store the helper functions in a reusable file.
 
+**Note:** Finished sketch its inside the tutorials library wrapper at:  
+*Examples -> Arduino_Pro_Tutorials -> Storage -> Creating a Flash-Optimised Key-Value Store -> FlashKeyValueStore*
```

Comment:
@marqdevx Why is this one in a `storage` folder? It's the only example that cannot be found directly by looking up the tutorial name.

Reply:
Fixed here https://github.com/arduino-libraries/Arduino_Pro_Tutorials/pull/1